testsuite/tests/hidden_includes/c3_1.ocamlc.reference

```diff
@@ -0,0 +1,4 @@
+File "libc/c3.ml", line 1, characters 8-11:
+1 | let x = A.x + 1
+            ^^^
+Error: Unbound module A
```

Comment:
Here's a suggestion for an improved error message:
```
Error: The module A is unbound because it is hidden. (See manual section ....)
```
Or 
```
Error: The module A cannot be referred to directly, because it is only found in a hidden (-H) include path.
(See manual section ...)
```

We would need to propagate how a particular path was found, but it seems like that should be straightforward.